tests/cross-product/storefront/scenarios/quick-order/shared-quick-order-scenario.js

```diff
@@ -0,0 +1,64 @@
+import { AbstractScenario } from '../../../../abstract-scenario.js';
+import { check } from 'k6';
+
+export class SharedQuickOrderScenario extends AbstractScenario {
+    async execute(productSku, quantity) {
+        const context = await this.browserHelper.getLoggedInUserContext();
+        const page = context.newPage();
+
+        try {
+            await this.openQuickOrderPage(page);
+            await this.searchForProduct(page, productSku);
+            await this.inputProductQuantity(page, quantity);
+            await this.createOrder(page);
+        } finally {
+            page.close();
+        }
+    }
+
+    async openQuickOrderPage(page) {
+        await page.goto(`${this.getStorefrontBaseUrl()}/en/quick-order`);
```

Comment:
Did you check if the data is output correctly for each such request (in the same way as when we use http.get)?